README.md

```diff
@@ -55,29 +55,41 @@ export PYTHONPATH=/sofa/build_dir/lib/python3/site-packages:$PYTHONPATH
 import sofagym
 ```
 
+### Test Script
+
+Running any of the available environments without RL training is possible using the script ```test_env.py```
```

Comment:
Maybe explain what it does if there is no training

Reply:
I updated it with further explanation